spec/unit/block_score/actions/retrieve_spec.rb

```diff
@@ -25,4 +25,18 @@
       end
     end
   end
+
+  describe '.included' do
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/83/files#r53864537